### PR TITLE
API Org App - Technical Test - Bien Jerico Cueto

### PR DESCRIPTION
**Pull request by bienjerico**
**To run the application:**
  - kindly install MongoDB and NodeJS
  - open terminal
  - enter "npm install" and wait to install all packages
  - run "npm start" in terminal to run the application
  - open any web browser then go to "http://localhost:3000"

**What design patterns did you use inside the application?**
I used Basic MEAN Full Stack Development.

**What libraries did you use?**
  MongoDB, ExpressJS, AngularJS, NodeJS, Bootstrap and JQuery

**Why did you use this library?**
  - MongoDB for database with mongoose framework to store my data
  - ExpressJS for NodeJS framework to simplify my codes
  - AngularJS for front-end development, because I don't want my page load so I use data binding
  - NodeJS for server
  - Bootstrap and JQuery for user interface because its easy to use.

**What major challenges did you encounter when making the app?**
  - Ofcourse time because I can only do the exam after office hours
  - My main language is PHP and I never use MEAN full stack in my work, so I have to self-study in a few days to create the system.

**If you had more time, what additional features would you like to add?**
  - Login feature like Facebook and Google.
  - Admin feature to consulidate all Notes and Todos inputed from guest users.
  - Google Analytics to see if its very useful.
  - For Todo I will add due date.
  - Validations

### CODE REVIEW

**Comment by zakame** [comment]
You committed `node_modules`?  That's not needed...

**Comment by bienjerico** [comment]
Hi @zakame,

 I already removed the node_modules folder.

Regards,
Bien

File: package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "mean-yoyo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
```
Comment (zakame) [inline package.json]: Missing `repository` field here, which fails npm:

```
~/api-org-app $ npm install
npm WARN mean-yoyo@1.0.0 No repository field.
```

Alternatively, you can mark the package as `private`...

**Comment by zakame** [comment]
@bienjerico thanks!

For the jquery/angular/bootstrap stuff, you didn't use `bower` to set them up?

**Comment by bienjerico** [comment]
@zakame Yes. I didn't use bower because I'm not yet familiar.

File: package.json (hunk 1 above)
Comment (bienjerico) [inline package.json]: @zakame I already marked the package as a private.  Can you try again?

**Pull request closed without merging**